.travis.yml

```diff
@@ -1,16 +1,12 @@
 language: android
 
-groovy:
```

Comment:
In practice, we are - that's how the bug surfaced in the first place!

The problem is that different versions expose slightly different APIs.  In this case, `Collections.forEach` didn't exist prior to 2.3.9.  Testing against an older version would (in theory) prevent regressions, though it seems like this never worked to begin with.